### ISSUE TITLE
Is Instagram still blocking?

### ISSUE THREAD

**Issue by Raspi123** [instabot-py/instabot.py#2477]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [ ] Bug
- [ ] Feature Request
- [ ] Technical Help (Please don't create an issue, join the facebook group or telegram channel)
- [x] Other

#### Environment
- [x] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] Are you running the code on your phone 
- [ ] If so, have you tried running the code on your local machine?

```
Include more details about the environment
Raspberry Pi
```

#### Operating System? (include version)
- [ ] macOS
- [ ] Windows
- [x] Rasbian Linux Raspberry PI (include flavour)
- [ ] Linux (include flavour)
- [ ] Android (include terminal)

```
include the output of 'uname -a'
not used
```

#### Python Version Requirement
- [x] I am using Python 3.6 or above

#### Exact Python Version?
```
Include here the response of 'python --version' or 'python3 --version'
3.4
```

#### Pip Version?
```
Include here the response of 'pip --version' or 'pip3 --version'
19.1.3
```

#### Instabot Version?
latest
```
Include here the response of 'instabot-py --version'
```

#### How you use the instabot
- [ ] instabot-py -c my-configfile.yml (my-configfile.yml is an example you may have a different file )
- [ ] instabot-py --login LOGIN --password PASSWORD
- [x] instabot-py 
- [ ] example.py (Deprecated, please install newer version) 

#### Description of your issue

Is Instagram still blocking people using this bot?
I was using this bot and got blocked a few month ago. Now I was wondering if it is safe to start it up again? 

```

```

#### Config

```

```

**Comment by mrklintscher** [comment]
Yes i used this Bot and Instagram blocks my account. I think it will not work anymore.

**Comment by noahwaldner** [comment]
Did you get a temporary or permanent block?

**Comment by xuhig** [comment]
omg

**Comment by danielpettifer** [comment]
IP gets blocked - Change IP, reduce frequency of adds/likes/etc

**Comment by mactrash** [comment]
Blocked IP and like after 5 likes and 8 follows same ip can login with app / web client without issue

**Comment by Raspi123** [comment]
Is someone who got blocked once running it again successfully?

**Comment by mactrash** [comment]
2nd test after unblock, use org / default config file and have no success to like / follow due to not meet the minimal no. of the default requirement... still got blocked... it mean the bot do 0 like and 0 follow but still got block.

**Comment by Raspi123** [comment]
I am wondering how they are detecting it..
Have noticed some people are again running bots on IG

**Comment by mrklintscher** [comment]
@nickgryg you closed it? What’s your workaround?

**Comment by nickgryg** [comment]
There are tons of closed issues about Instagram ban in this repo. Please find them and read their solutions and not just tick `I have searched for other issues with the same problem` when create new issue. 

BTW, How can we provide a piece of advice here if we don't even know instabot version you use, configuration etc?

**Comment by mrklintscher** [comment]
The latest one isn’t working because of the latest insta ai! Yes we have one because without fixing it you shouldn’t use it! But i do a new issue to become clearly!
